sys/include/net/netstats.h

```diff
@@ -41,6 +50,16 @@ typedef struct {
     uint32_t rx_bytes;          /**< received bytes */
 } netstats_t;
 
+/**
+ * @brief   Get a string description for @ref netstats_module_t, for debugging
```

Comment:
Hm, well, actually I cannot do it - only for param and return, because otherwise brief would exceed the 80 character limit and would require a linebreak (which is not simple with doxygen).